module-2/backend/week_three.md

```diff
@@ -9,15 +9,50 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 
 ### Questions
 
-1. What is the entry at the command line to create a new rails app?
-2. What do Models generally inherit from in rails?
+1. What is the entry at the command line to create a new rails app? 
+  rails new whatever the app name is (you can specify more things in this command line like what sql database you want to use etc.)
+2. What do Models generally inherit from in rails? 
+  ApplicationRecord
 3. What do Controllers generally inherit from in a rails project?
+  ApplicationController
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  resources :horses, only[:show]
 5. What rake task is useful when looking at routes, and what information does it give you?
+  rake routes it gives you all the available path names, url names, and what method it needs in the controller
 6. What is an example of a route helper? When would you use them?
+  in our routes file we can specify what we want the actual route name to be, so if we wanted to do a login we would set it as a login as opposed to it being create(it's more indicative of what we are trying to do and that way it is easier for users to use)
```

Comment:
This is all good knowledge, but the route helper is actually something like `horses_path`.